### PR TITLE
Responding to issues 35

### LINKED ISSUE #35: "RequireSchemaAuthz = public" is set when CELL ACL is changed

### PR DESCRIPTION
**Pull request by csKyou**
- Fix to not change RequireSchemaAuthz
Close #35

### CODE REVIEW

File: src/js/cellAcl.js [hunk 1]
```diff
@@ -108,7 +108,10 @@ cellAcl.prototype.implementCellACL = function (arrCellACLRolePrivilegeList) {
 	var objAce = null;
 	var objJAcl = new _pc.Acl();
 	var objPrincipal = new _pc.Principal();
-	objJAcl.setRequireSchemaAuthz("public");
+	var rolePrivList = this.getCellACLSettings();
+	if (rolePrivList.schemaAuthz != "null") {
```
Comment (dixonsiu) [inline src/js/cellAcl.js]: Would it be better to set a default value when rolePrivList.schemaAuthz is null?

Comment (dixonsiu) [inline src/js/cellAcl.js]: Should move logic to the function that uses schemaAuthz since it is already filtering out ""/null/undefined.

**Review by dixonsiu — CHANGES_REQUESTED** [review]

File: src/js/cellAcl.js (hunk 1 above)
Comment (csKyou) [reply src/js/cellAcl.js]: Default value is not required.
Inherit existing settings.

Comment (csKyou) [reply src/js/cellAcl.js]: corresponded.

**Review by dixonsiu — APPROVED** [review]

**Comment by dixonsiu** [comment]
Closes #14

**Pull request merged**